cvat/settings/base.py

```diff
@@ -217,6 +218,7 @@ def add_ssh_keys():
     # FIXME
     # 'corsheaders.middleware.CorsPostCsrfMiddleware',
     'django.contrib.auth.middleware.AuthenticationMiddleware',
+    'crum.CurrentRequestUserMiddleware',
```

Comment:
We will need to remove the extension. It isn't well supported. Let's think how to do that.